c/include/libsbp/v4/sbp_msg.h

```diff
@@ -1855,6 +1861,8 @@ static inline size_t sbp_message_encoded_len(sbp_msg_type_t msg_type,
 static inline int sbp_message_cmp(sbp_msg_type_t msg_type, const sbp_msg_t *a,
                                   const sbp_msg_t *b) {
   switch (msg_type) {
+    case SbpMsgInvalid:
+      return 0;
```

Comment:
This is in `sbp_message_cmp`; return 0 to match the behavior of its `default` case